core.d.ts

```diff
@@ -355,10 +355,21 @@ declare namespace core {
 	*/
 	const mimeTypes: Set<core.MimeType>;
 
+	/**
+	 * Stream options.
+	 */
+	interface IStreamOptions {
+		/**
+		 *  Sample size in bytes.
```

Comment:
It should document the default value using standard docs annotation.

Reply:
> Not done: [#434 (comment)](https://github.com/sindresorhus/file-type/pull/434#discussion_r674708584)

Fixed